### PR TITLE
Minor vehicle burn simplifications

### PR DESCRIPTION
**Pull request by elffjs**
# Proposed Changes

### CODE REVIEW

File: internal/controllers/nft_controller.go [hunk 1]
```diff
@@ -701,17 +701,15 @@ func (udc *UserDevicesController) GetBurnDevice(c *fiber.Ctx) error {
 
 	vehicleNFT, err := models.VehicleNFTS(
 		models.VehicleNFTWhere.TokenID.EQ(tid),
-		qm.Load(qm.Rels(models.VehicleNFTRels.UserDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.MintRequest)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.VehicleTokenAftermarketDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.VehicleTokenSyntheticDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.UserDevice, models.UserDeviceRels.UserDeviceAPIIntegrations)),
+		qm.Load(models.VehicleNFTRels.UserDevice),
```
Comment (elffjs) [inline internal/controllers/nft_controller.go:704]: Remove some loads we don't seem to need.

File: internal/controllers/nft_controller.go [hunk 2]
```diff
@@ -701,17 +701,15 @@ func (udc *UserDevicesController) GetBurnDevice(c *fiber.Ctx) error {
...
+		qm.Load(models.VehicleNFTRels.VehicleTokenAftermarketDevice),
+		qm.Load(models.VehicleNFTRels.VehicleTokenSyntheticDevice),
 	).One(c.Context(), tx)
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
-			return fiber.NewError(fiber.StatusNotFound, "No command-capable integrations found for this vehicle.")
+			return fiber.NewError(fiber.StatusNotFound, "No vehicle NFT with that token id.")
```
Comment (elffjs) [inline internal/controllers/nft_controller.go:710]: Error message maybe copied from the vehicle NFT command endpoints.

File: internal/controllers/nft_controller.go [hunk 3]
```diff
@@ -839,6 +836,10 @@ func (udc *UserDevicesController) PostBurnDevice(c *fiber.Ctx) error {
 func (udc *UserDevicesController) checkDeviceBurn(ctx context.Context, vehicleNFT *models.VehicleNFT, userID string) (registry.BurnVehicleSign, *pb.User, error) {
 	var bvs registry.BurnVehicleSign
 
+	if vehicleNFT.R.BurnRequest != nil && vehicleNFT.R.BurnRequest.Status != models.MetaTransactionRequestStatusFailed {
```
Comment (elffjs) [inline internal/controllers/nft_controller.go:839]: Attempt to stop double-submit.

File: internal/controllers/nft_controller.go [hunk 4]
```diff
@@ -857,10 +858,8 @@ func (udc *UserDevicesController) checkDeviceBurn(ctx context.Context, vehicleNF
 	}, user, nil
 }
 
-// BurnRequest contains the user's signature for the mint request as well as the
-// NFT image.
+// BurnRequest contains the user's signature for the burn request.
```
Comment (elffjs) [inline internal/controllers/nft_controller.go:861]: We already have token id in the path.

File: internal/controllers/nft_controller.go (hunk 1 above)
Comment (elffjs) [inline internal/controllers/nft_controller.go:704]: Don't need `Rels` if it's just one step.

File: internal/controllers/nft_controller.go [hunk 5]
```diff
@@ -701,17 +701,15 @@ func (udc *UserDevicesController) GetBurnDevice(c *fiber.Ctx) error {
 
 	vehicleNFT, err := models.VehicleNFTS(
 		models.VehicleNFTWhere.TokenID.EQ(tid),
-		qm.Load(qm.Rels(models.VehicleNFTRels.UserDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.MintRequest)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.VehicleTokenAftermarketDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.VehicleTokenSyntheticDevice)),
-		qm.Load(qm.Rels(models.VehicleNFTRels.UserDevice, models.UserDeviceRels.UserDeviceAPIIntegrations)),
```
Comment (elffjs) [inline internal/controllers/nft_controller.go:708]: Remove some unused joins.

**Review by Allyson-English — APPROVED** [review]

**Pull request merged**